students/morgan/mailprog/mailprog/functions.py

```diff
@@ -0,0 +1,80 @@
+from mailprog.dataModels import Donor
+
+def list_donors(DB):
```

Comment:
if these were all in a class (maybe the Storage class), then they would all have access to the DB.